src/test/java/org/folio/rest/camunda/utility/TestUtility.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.rest.camunda.utility;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import org.springframework.http.ResponseEntity;
+
```

Comment:
Could you add Javadocs describing the purpose of this test utility class?

Reply:
Done.